src/contexts/Guilds/orbis.tsx

```diff
@@ -0,0 +1,14 @@
+import { createContext, useRef } from 'react';
+import { Orbis } from '@orbisclub/orbis-sdk';
+
+export const OrbisContext = createContext(null);
+
+export const OrbisProvider = ({ children }) => {
+  let orbis = useRef(new Orbis());

```

Comment:
I don't think we need to use the hook `useRef` now since the state is in context instead and will not be updated on render (which was the problem before and thus why I resolved to using `refs`.